### ISSUE TITLE
Settings.vala:87: Could not connect: No such file or directory ** (com.github.phase1geo.minder:2)

### ISSUE THREAD

**Issue by robertoswald** [flathub/com.github.phase1geo.minder#6]
Fresh Ubuntu 20.04, fresh install via flathub.
Minder is not starting with this error message:
```
erwin@erwin-VirtualBox:~$ flatpak run com.github.phase1geo.minder 
Gtk-Message: 13:34:59.670: Failed to load module "canberra-gtk-module" 
Gtk-Message: 13:34:59.670: Failed to load module "canberra-gtk-module" ** (com.github.phase1geo.minder:2): 
CRITICAL **: 13:35:00.328: Settings.vala:87: Could not connect: No such file or directory ** (com.github.phase1geo.minder:2): CRITICAL **: 13:35:00.328: Settings.vala:107: Could not connect: No such file or directory
```

**Comment by tim77** [comment]
Thanks for reporting. Should be fixed in 1.12.4 https://flathub.org/builds/#/builders/32/builds/49819. Already updated on Flathub.

**Comment by robertoswald** [comment]
Hi,

I can confirm, works now with 1.12.4.
Thanks for fixing.
